addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -131,7 +131,7 @@ var RunningTourActionHelper = core.Class.extend({
 
         var toCenter = $to.offset();
         toCenter.left += $to.outerWidth()/2;
-        toCenter.top += $to.outerHeight()/2;
+        toCenter.top += $to.outerHeight()/3;
```

Comment:
Why ?

Reply:
To drop snippets little upper (Otherwise it will be dropped under footer, which was not possible before this commit)

Reply:
Yep few tours were broken. I'll revert this change, and fix the broken tours 👍

Reply:
Improved the tours to drag snippets under `#wrap` where needed